app/components/EventsDashboard.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+import { NextFetchEvent } from "next/server";
+import React, { useEffect, useState } from "react";
+
+interface Event {
+    name: string;
+    count: number;
+}
+async function fetchEvents() {
+    const response = await fetch('https://mixpanel.com/api/2.0/events/names', {
+        method: 'GET',
```

Comment:
@JNicolao what are all these fees under public folder?

Reply:
I created a component in the public/app folder named EventsDashboard.tsx.